data/themes/default/style.css

```diff
@@ -946,6 +946,14 @@ lmms--gui--SidInstrumentView lmms--gui--Knob {
 	qproperty-lineWidth: 2;
 }
 
+lmms--gui--SlicerTUI lmms--gui--Knob {
```

Comment:
I think this should be applied to the classic theme too.

Reply:
Done